frontend/views/containers/chatroom/ChatNav.vue

```diff
@@ -1,58 +1,18 @@
 <template lang='pug'>
 .c-chatnav
-  template(slot='actions')
-    menu-parent
-      menu-trigger.is-icon
-        i.icon-ellipsis-v
-
-      menu-content.c-actions-content
-        ul
-          menu-item(tag='button' itemid='hash-1' icon='heart')
-            i18n Settings
-          menu-item(tag='button' itemid='hash-2' icon='heart' hasdivider='')
-            i18n Mute chat
-          menu-item(tag='button' itemid='hash-3' icon='heart')
-            i18n Report a problem
-
-  .c-chatnav-body
-    slot
-
```

Comment:
@Silver-IT is Sebin right that this is no longer needed?

Reply:
@Silver-IT @taoeffect 
(As you might just know by looking at the code)`template(slot='actions')` here is silently ignored and removed from the DOM at runtime (Apparently there used to be a child component in here to which this `slot` was passed). so this code practically becomes,
`.c-chatnav`
`  .c-chatbody`
`     slot`
here. Then I decided to just turn it into
`.c-chatnav`
`  slot`
As it doesn't need to wrapped twice. Thanks.

Reply:
Currently I'm not sure. I will review it when I get back.

Reply:
@SebinSong, I think that's not needed. Maybe that part was copied from another component when it was created.
Actually, I think there could be some more unused lines of codes in chat part such as Messages.vue page and it's route. I could guess the previous developer was going to use Messages.vue for the direct messages.